sources/web/datalab/polymer/components/datalab-data/datalab-data.ts

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/// <reference path="../input-dialog/input-dialog.ts" />
+/// <reference path="../item-list/item-list.ts" />
+
+interface Collection {
+  name : string;
+  status : string;
+  datasets? : Array<Dataset>;
+}
+
+interface Dataset {
```

Comment:
Yes, we definitely need to resolve the language here! This confused me for a few seconds. :)